[2024-05-26T02:04:47.950+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-26T02:04:47.975+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_catedra_airflow.second-step-el scheduled__2024-05-25T03:25:00+00:00 [queued]>
[2024-05-26T02:04:47.984+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_catedra_airflow.second-step-el scheduled__2024-05-25T03:25:00+00:00 [queued]>
[2024-05-26T02:04:47.984+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-26T02:04:47.999+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): second-step-el> on 2024-05-25 03:25:00+00:00
[2024-05-26T02:04:48.006+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=355) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-26T02:04:48.008+0000] {standard_task_runner.py:63} INFO - Started process 357 to run task
[2024-05-26T02:04:48.008+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DAG_catedra_***', 'second-step-el', 'scheduled__2024-05-25T03:25:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/dag_clase.py', '--cfg-path', '/tmp/tmpw7fnua8s']
[2024-05-26T02:04:48.009+0000] {standard_task_runner.py:91} INFO - Job 80: Subtask second-step-el
[2024-05-26T02:04:48.055+0000] {task_command.py:426} INFO - Running <TaskInstance: DAG_catedra_airflow.second-step-el scheduled__2024-05-25T03:25:00+00:00 [running]> on host da1b21f42040
[2024-05-26T02:04:48.159+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_catedra_***' AIRFLOW_CTX_TASK_ID='second-step-el' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T03:25:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-25T03:25:00+00:00'
[2024-05-26T02:04:48.169+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-26T02:04:48.176+0000] {logging_mixin.py:188} INFO - The connection to the DB was succesful
[2024-05-26T02:04:48.190+0000] {logging_mixin.py:188} INFO - Query completed
[2024-05-26T02:04:48.195+0000] {logging_mixin.py:188} INFO - execute_query_result_set()
[2024-05-26T02:04:48.201+0000] {logging_mixin.py:188} INFO - The connection to the DB was succesful
[2024-05-26T02:04:48.207+0000] {logging_mixin.py:188} INFO - Inserts were comitted
[2024-05-26T02:04:48.208+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-26T02:04:48.208+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-26T02:04:48.217+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=DAG_catedra_***, task_id=second-step-el, run_id=scheduled__2024-05-25T03:25:00+00:00, execution_date=20240525T032500, start_date=20240526T020447, end_date=20240526T020448
[2024-05-26T02:04:48.262+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-26T02:04:48.280+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-26T02:04:48.285+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
